src/date-picker/date-picker-inline.jsx

```diff
@@ -0,0 +1,56 @@
+const React = require('react');
+const Paper = require('../paper');
+
+const DatePickerInline = React.createClass({
+
+  propTypes: {
+    open: React.PropTypes.bool,
+  },
+
+  getDefaultProps: function() {
+    return {
+      open: false,
+    };
+  },
+
+  render() {
+    let {
+      style,
+      actions,
+      open,
+      ...other,
+    } = this.props;
+
+    let styles = {
+      actions: {
+        marginRight: 8,
+        textAlign: 'right',
+      },
+      container: {
+        zIndex: 100,
+        width: '100%',
+        position: 'relative',
+        display: (open === true ? 'block' : 'none'),
```

Comment:
Instead of using the display property, why not doing in the render method?
This would prevent to render the Calander when it's hidden. Though?